src/main/sensors/voltage.c

```diff
@@ -134,11 +134,15 @@ static const uint8_t voltageMeterAdcChannelMap[] = {
 #endif
 };
 
+#if !defined(ADCVREF)
+#define ADCVREF 3300
```

Comment:
This should not be defined twice, but rather should be defined in a header file. Also it should have a self-documenting name, eg: `ADC_VOLTAGE_REFERENCE_MV`